subscriptions/templatetags/seed.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+import seed_stage_based_messaging
+
+register = template.Library()
```

Comment:
This template tag library should eventually be moved to an external package but at the moment it doesn't justify the overhead.